src/test/kotlin/TestProject/TestHelloWorld.kt

```diff
@@ -0,0 +1,12 @@
+package TestProject
```

Comment:
The package structure in the `test` tree should mirror the package structure in the `main` tree. I think your package was called `FinalProject?, right?